composer.json

```diff
@@ -197,7 +197,6 @@
     },
     "conflict": {
         "symfony/symfony": "*",
-        "doctrine/orm": "2.12.0 || >=2.16.0",
```

Comment:
same conflict should be removed in the `packages/framework/composer.json`

Reply:
fixed